.env.example

```diff
@@ -1,6 +1,11 @@
 # Gemfile
 GEM_PG_VERSION=1.2.2
 
+# docker-compose.yml - postgres
+POSTGRES_USER=gtt
+POSTGRES_PASSWORD=gtt
+POSTGRES_DB=gtt
+
```

Comment:
@iosefa (CC: @dkastl)  
Because of the priority of these params, I set these values above smtp settings.

Having default values reason is the following.

> - Keep original params (all `gtt`) as example (to ensure that just copying `.env.example` to `.env` works without problem).

Reply:
@iosefa Well, I will merge this now before updating RedMica version.